cranelift/tests/bugpoint_test_expected.clif

```diff
@@ -73,4 +42,12 @@ block117:
     v987 = iconst.i64 0
     call fn0(v0, v105, v1052, v883, v829, v987, v951, v842)
     trap user0
+
+block120:
+    v1052 = iconst.i16 0
```

Comment:
Maybe we should try to move instructions to the entry block in a new mutator. I opened #1540 for this.